docs/config.md

```diff
@@ -139,7 +142,7 @@ profile_models: <comma-delimited-string-list>
 [ triton_docker_labels: <dict> ]
 
 # How Model Analyzer will launch triton. It should
-# be either "docker", "local", or "remote".
+# be either "docker", "local", "remote" or "C_API".
```

Comment:
How about using lower case `c_api` to be more consistent with perf_analyzer?

Reply:
Will do